src/dapps/ScheduleTransaction/containers/ScheduleView.vue

```diff
@@ -0,0 +1,854 @@
+          and another one for token transfer approval.
+        </b-alert>
+
+        <div
+          :class="[
+            validInputs ? '' : 'disabled',
+            'submit-button large-round-button-green-filled'
+          ]"
+          @click="scheduleTx"
+        >
+          Schedule Transaction
+        </div>
+      </div>
+    </div>
+    <a
+      href="https://blog.chronologic.network/announcing-the-ethereum-alarm-clock-chronologic-partnership-b3d7545bea3b"
+      target="_blank"
+      rel="noopener noreferrer"
+      class="eac-logo"
+    />
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { EAC, Util } from '@ethereum-alarm-clock/lib';
+import BigNumber from 'bignumber.js';
+import { Datetime } from 'vue-datetime';
+import 'vue-datetime/dist/vue-datetime.css';
+import moment from 'moment';
+import 'moment-timezone';
+import * as unit from 'ethjs-unit';
+import { Toast } from '@/helpers';
+
+import BackButton from '@/layouts/InterfaceLayout/components/BackButton';
+import CurrencyPicker from '../../../layouts/InterfaceLayout/components/CurrencyPicker';
+import StandardInput from '@/components/StandardInput';
+import StandardDropdown from '@/components/StandardDropdown';
+import Blockie from '@/components/Blockie';
+import { ERC20 } from '@/partners';
+import {
+  calcSchedulingTotalCost,
+  canBeConvertedToWei,
+  EAC_SCHEDULING_CONFIG,
+  estimateBountyForGasPrice
+} from '../ScheduleHelpers';
+
+export default {
+  name: 'ScheduleView',
+  components: {
+    'back-button': BackButton,
+    'currency-picker': CurrencyPicker,
+    'standard-input': StandardInput,
+    'standard-dropdown': StandardDropdown,
+    'datetime-picker': Datetime,
+    blockie: Blockie
+  },
+  props: {
+    tokensWithBalance: {
+      type: Array,
+      default: function() {
+        return [];
+      }
+    }
+  },
+  data() {
+    return {
+      eac: null,
+      advancedExpand: false,
+      advancedTimeBounty: false,
+      hexAddress: '',
+      address: '',
+      isValidAddress: false,
+      amount: '0',
+      gasLimit: EAC_SCHEDULING_CONFIG.FUTURE_GAS_LIMIT,
+      futureGasLimit: EAC_SCHEDULING_CONFIG.FUTURE_GAS_LIMIT,
+      minGasLimit: 0,
+      futureGasPrice: '1',
+      minGasPrice: 0.1,
+      data: '',
+      datetime: '',
+      currentBlockNumber: '',
+      selectedBlockNumber: '',
+      timeBountyPresets: EAC_SCHEDULING_CONFIG.TIME_BOUNTY_DEFAULTS,
+      timeBounty: EAC_SCHEDULING_CONFIG.TIME_BOUNTY_DEFAULTS[0],
+      windowSize: 10,
+      supportedModes: EAC_SCHEDULING_CONFIG.SUPPORTED_MODES,
+      selectedMode: EAC_SCHEDULING_CONFIG.SUPPORTED_MODES[0],
+      deposit:
+        EAC_SCHEDULING_CONFIG.TIME_BOUNTY_DEFAULTS[0] *
+        EAC_SCHEDULING_CONFIG.BOUNTY_TO_DEPOSIT_MULTIPLIER,
+      ethPrice: new BigNumber(0),
+      selectedTimeZone: moment.tz.guess(),
+      selectedCurrency: '',
+      showTokenTransferNotification: true,
+      amountInputOptions() {
```

Comment:
These functions that returns the options might be better suited in the `computed` property so whenever the `this` instances inside updates, it also updates instead of being called only once.